components/automate-cli/cmd/chef-automate/version.go

```diff
@@ -58,6 +58,12 @@ func printServerVersion() error {
 			"Server version cannot be listed because this command was not run as root. Re-run this command as root to see full server version information.",
 		)
 	}
+	
+	// Check for bastion
+	if isA2HARBFileExist() {
+		writer.Bodyf("Server Build : For getting chef-automate version, Please login to chef-automate and use command chef-automate version ")
```

Comment:
Why are we adding "server Build " key word in message any specific reason for that?. Ideally we should just print the message and suggested mechanism for getting the version.

Reply:
Just to maintain uniformity of how the output is coming , chef-automate version command output will show cli build first and so message should be specific to only server build .

Reply:
On bastion machine
![Screenshot from 2022-01-27 14-59-24](https://user-images.githubusercontent.com/60912283/151331159-bf5af6c6-9a83-477b-a914-3f54a666a79b.png)
on automate Machine
![Screenshot from 2022-01-27 13-35-14](https://user-images.githubusercontent.com/60912283/151331201-151af173-ac98-4253-be01-eb2ff3f1a6e7.png)